src/js/pensions/config/form.js

```diff
@@ -748,22 +765,29 @@ const formConfig = {
             dependents: {
               'ui:options': {
                 expandUnder: 'view:hasDependents',
-                viewField: DependentField
+                viewField: DependentField,
               },
               items: {
-                relationship: {
+                dependentRelationship: {
                   'ui:title': 'Relationship to Veteran',
                   'ui:widget': 'radio',
                   'ui:options': {
                     labels: relationshipLabels
                   }
                 },
-                fullName: fullNameUI,
-                childDateOfBirth: _.merge(currentOrPastDateUI('Date of birth'), {
+                fullName: _.merge(fullNameUI, {
                   'ui:options': {
-                    hideIf: (form, index) => _.get(['dependents', index, 'relationship'], form) !== 'child'
+                    expandUnder: 'dependentRelationship',
```

Comment:
Have made this faux conditional to allow `childDateOfBirth` to use `expandUnder` rather than `hideIf` to ensure proper ordering, as `currentOrPastDateUI` is behaving in an unexpected way (possibly due to `updateschema` logic elsewhere).